tethysext/atcore/services/map_manager.py

```diff
@@ -528,10 +532,10 @@ def generate_custom_color_ramp_divisions(self, min_value, max_value, num_divisio
         b = max_val - (m * max_div)
 
         for i in range(min_div, max_div + 1):
-            if (max_value - min_value) / num_divisions <= 2:
-                divisions['{}{}'.format(prefix, i)] = "{0:.5f}".format(m * i + b)
-            else:
-                divisions['{}{}'.format(prefix, i)] = ceil(m * i + b)
+            divisions[f'{prefix}{i}'] = f"{(m * i + b):.{value_precision}f}"
```

Comment:
BTW, look at this beauty. You can have nested variables within an f string.

Reply:
Nice